lib/plugins/aws/package/compile/vpc/lib/subnets.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const { isObject, isEmpty } = require('lodash');
+const { SUBNET_TYPES, SUBNET_CONFIG } = require('./utils/constants');
+const { flatMap } = require('./utils/functions');
+const getSubnetTemplate = require('./templates/subnet');
+
+function compileSubnets(subnetConfigs) {
+  const { Resources } = this.serverless.service.provider.compiledCloudFormationTemplate;
+  const types = [SUBNET_TYPES.PRIVATE, SUBNET_TYPES.PUBLIC];
+  // NOTE: previously we loaded the supported regions from `./utils/constants`
+  // we keep the `flatMap` code intact to allow multi-region azs later on
+  const availabilityZones = flatMap(region => [`${region}a`], [this.options.region]);
```

Comment:
I'm confused on the `availabilityZones` part -- does this only use the `a` availability zone for the region? The docs say that the default config creates subnets in both `a` and `b` AZs for the given region -- how is that accomplished? The test outputs only show `a` from what I can tell.

Also, FWIW, it might be useful to use the [`GetAZs`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-getavailabilityzones.html) intrinsic function in CloudFormation for this.

Reply:
Similar to what my plugin does, but would it be possible to make a call to the AWS API to get the available AZs in a region?  Sometimes they can be marked as unavailable so I'm not sure we should rely on looking for `a` or `b` in the naming convention.